src/Object/Post.php

```diff
@@ -337,6 +338,7 @@ public function getTemplateData($conv_responses, $thread_level = 1)
 		$tmp_item = [
 			'template'        => $this->getTemplate(),
 			'type'            => implode("", array_slice(explode("/", $item['verb']), -1)),
+			'suppress_tags'   => Config::get('system', 'suppress_tags'),
```

Comment:
Shouldn't we set ```false``` as default value?

Reply:
Not necessarily, the default value would be `null` which truthiness is `false` and is enough for how we use it.